.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     #### expecting it in the form of
     ####   /go/src/github.com/circleci/go-tool
     ####   /go/src/bitbucket.org/circleci/go-tool
-    working_directory: /go/src/github.com/ypgao1/mattermost-plugin-bigbluebutton
+    working_directory: /go/src/github.com/blindsidenetworks/mattermost-plugin-bigbluebutton
```

Comment:
Why don't you better use a relative path? This makes the code work only for this particular repo
Not sure if this would do:

```
working_directory: ~/
```

Or this one
```
working_directory: ~/repo
```

Reply:
Its a Golang thing:
```
    #### TEMPLATE_NOTE: go expects specific checkout path representing url
    #### expecting it in the form of
    ####   /go/src/github.com/circleci/go-tool
```

Reply:
It has to be a way. It may be something like this
```
working_directory: /go/src/github.com/{{ .Environment.CIRCLE_USERNAME }}/mattermost-plugin-bigbluebutton
```
But no worries, I'll fix it later